docs/riff.md

```diff
@@ -1,26 +1,28 @@
 ## riff
 
-Commands for creating and managing function resources
+
 
 ### Synopsis
 
-riff is for functions.
 
-riff is a CLI for functions on Knative.
-See https://projectriff.io and https://github.com/knative/docs
 
 ### Options
 
 ```
-  -h, --help   help for riff
+      --config string       config file (default is $HOME/.riff.yaml)
+  -h, --help                help for riff
+      --kubeconfig string   kubectl config file (default is $HOME/.kube/config)
 ```
 
 ### SEE ALSO
 
-* [riff completion](riff_completion.md)	 - Generate shell completion scripts
-* [riff function](riff_function.md)	 - Interact with function related resources
-* [riff namespace](riff_namespace.md)	 - Manage namespaces used for riff resources
-* [riff service](riff_service.md)	 - Interact with service related resources
-* [riff system](riff_system.md)	 - Manage system related resources
-* [riff version](riff_version.md)	 - Print version information about riff
+* [riff application](riff_application.md)	 - 
+* [riff completion](riff_completion.md)	 - 
+* [riff credential](riff_credential.md)	 - 
+* [riff function](riff_function.md)	 - 
+* [riff requestprocessor](riff_requestprocessor.md)	 - 
+* [riff stream](riff_stream.md)	 - 
+* [riff streamprocessor](riff_streamprocessor.md)	 - 
+* [riff version](riff_version.md)	 - 
 
+###### Auto generated by spf13/cobra on 6-May-2019
```

Comment:
do we want this mention? how come it was not generated before?

Reply:
We'll need to kill that otherwise CI will fail. I'm not sure why it's there because we're using the same version of cobra and the same code to generate docs, maybe it's an internal library that version bumped.

Reply:
`cmd.DisableAutoGenTag = true`

Reply:
done